src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -142,7 +165,39 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     scheduledInterviewDao.create(
         ScheduledInterview.create(-1, interviewRange, interviewerId, intervieweeId));
 
-    // Since an interview was scheduled, both parties' availabilities must be updated
+    HashMap<String, String> emailedDetails = new HashMap<String, String>();
+    String interviewId =
+        String.valueOf(
+            scheduledInterviewDao
+                .getScheduledInterviewsInRangeForUser(
+                    intervieweeId, interviewRange.start(), interviewRange.end())
+                .get(0)
+                .id());
+    String intervieweeFeedbackLink =
+        String.format(
+            "http://interview-me-step-2020.appspot.com/feedback.html?interview=%s&role=interviewee",
+            interviewId);
+    String interviewerFeedbackLink =
+        String.format(
+            "http://interview-me-step-2020.appspot.com/feedback.html?interview=%s&role=interviewer",
+            interviewId);
+    emailedDetails.put("{{formatted_date}}", getEmailDateString(interviewRange));
+    emailedDetails.put("{{interviewer_first_name}}", getFirstName(interviewerId));
+    emailedDetails.put("{{interviewee_first_name}}", getFirstName(intervieweeId));
+    emailedDetails.put("{{form_link}}", intervieweeFeedbackLink);
+    emailedDetails.put("{{position}}", position);
```

Comment:
Can we format this String better? Software Engineer instead of SOFTWARE_ENGINEER.

Reply:
Done